packages/libxml2.rb

```diff
@@ -58,10 +63,12 @@ def self.check
     # Check https://mail.gnome.org/archives/xml/2010-April/msg00010.html for details.
     system 'rm', 'test/ebcdic_566012.xml'
 
+    # system "DESTDIR=#{CREW_DEST_DIR} #{CREW_NINJA} -C builddir check"
     system 'make', 'check'
   end
 
   def self.install
+    # system "DESTDIR=#{CREW_DEST_DIR} #{CREW_NINJA} -C builddir install"
```

Comment:
Can we just remove the commented lines for the cmake build?

Reply:
That makes sense.